script/run-axe.sh

```diff
@@ -3,6 +3,10 @@ set -euo pipefail
 
 # start a web server in the background and tear it down when exiting
 ./script/serve.sh public &
+
+# Wait for the python server to launch since we're launching it async
+sleep 3
```

Comment:
how would you feel about waiting on `lsof` or `curl` to test the port instead of waiting a set amount of time? Better or worse?

Reply:
For sure better, I was lazy!